snippets/getColonTimeFromDate.md

```diff
@@ -0,0 +1,11 @@
+### getColonTimeFromDate
+
+Formats hour, minute, and second time integers from Date into stringified colon representation. Formats hour integer to 12 hour clock and maintains digit lengths to match (HH:MM:SS).
```

Comment:
Hey @Emgo-Dev 👋 😄 

Thanks for taking the time out to submit this PR

I believe `new Date()` by default returns 24 hrs & not 12

Reply:
Yes it does which detracts from the simplicity of just slicing the colon time from the ```Date``` itself, because formatting the hour would require more slicing work.

It comes down to the preferences of the repo. Slicing a provided string serves little educational purpose as well, where constructing a string has more to look at.